llm/tests/compare_hf_v_mosaic_gpt.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2022 MosaicML Benchmarks authors
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import warnings
+
+import torch
+from omegaconf import OmegaConf as om
+from composer.utils import reproducibility
+
+from main import build_composer_model
```

Comment:
I think this path has changed with the latest changes on main, should be something like `from src.model_registry import COMPOSER_MODEL_REGISTRY`, I think.